src/types-and-traits.rst

```diff
@@ -2637,6 +2637,28 @@ A :t:`higher-ranked trait bound` is a :t:`bound` that specifies an infinite
 list of :t:`[bound]s` for all possible :t:`[lifetime]s` specified by the
 :s:`ForGenericParameterList`.
 
+:dp:`fls_J9DEsd06Ttu9`
+An :t:`implied bound` is a :t:`bound` that is not expressed in syntax, but is
+is the byproduct of relations between the :t:`[generic parameter]s` of
```

Comment:
between `lifetime parameters` and parameters and return types for functions 
between `lifetime parameters`  and fields for abstract data types